### DISCUSSION TITLE
[Bug] Unable to convert issues to discussions

### DISCUSSION THREAD

**Discussion by marlonrichert** [community/community#3356] [Discussions]
Given that I have several open issues labeled "question" at https://github.com/marlonrichert/zsh-snap/labels/question

When 
1. I go to https://github.com/marlonrichert/zsh-snap/labels and for the "question" label click "Convert issues"
2. I choose "Q&A".
3. I click " I understand, convert these issues to discussions. "

Then
* I expect these issues to be converted to discussions.
* BUT actually, nothing happens. The issues are still open and unlocked, and no new discussions have appeared.

**Comment by PranamBhat** [comment]
Hey!

It will be under [https://github.com/marlonrichert/zsh-snap/discussions](https://github.com/marlonrichert/zsh-snap/discussions)

Just refresh the page and go to "Discussions" tab.

Have a nice day!

**Reply by marlonrichert** [reply]
I’m reporting a bug, not asking a question.

As you can see here https://github.com/marlonrichert/zsh-snap/issues?q=label%3Aquestion+
I have 7 issues labeled “question”. 1 was closed when I converted it to a discussion. The other 6 open issues, however, I have not been able to convert. You will not find those at https://github.com/marlonrichert/zsh-snap/discussions

**Comment by marlonrichert** [comment]
@evi-liu Is this a known bug? Is there a workaround?
